content/en/docs/setup/independent/high-availability.md

```diff
@@ -10,513 +10,217 @@ content_template: templates/task
-
-1. Install keepalived, e.g. using your distribution's package manager. The configuration shown here works with version `1.3.5` but is expected to work with may other versions. Make sure to have it enabled (chkconfig, systemd, ...) so that it starts automatically when the respective node comes up.
-
-2. Create the following configuration file _/etc/keepalived/keepalived.conf_ on all master nodes:
-     ```none
-     ! Configuration File for keepalived
-     global_defs {
-      router_id LVS_DEVEL
-     }
-
-     vrrp_script check_apiserver {
-      script "/etc/keepalived/check_apiserver.sh"
-      interval 3
-      weight -2
-      fall 10
-      rise 2
-     }
-
-     vrrp_instance VI_1 {
-        state <STATE>
-        interface <INTERFACE>
-        virtual_router_id 51
-        priority <PRIORITY>
-        authentication {
-            auth_type PASS
-            auth_pass 4be37dc3b4c90194d1600c483e10ad1d
-        }
-        virtual_ipaddress {
-            <VIRTUAL-IP>
-        }
-        track_script {
-            check_apiserver
-        }
-     }
-     ```
-
-    In the section `vrrp_instance VI_1`, change few lines depending on your setup:
-
-    * `state` is either `MASTER` (on the first master nodes) or `BACKUP` (the other master nodes).
-    * `interface` is the name of an existing public interface to bind the virtual IP to (usually the primary interface).
-    * `priority` should be higher for the first master node, e.g. 101, and lower for the others, e.g. 100.
-    * `auth_pass` use any random string here.
-    * `virtual_ipaddresses` should contain the virtual IP for the master nodes.
-
-3. Install the following health check script to _/etc/keepalived/check_apiserver.sh_ on all master nodes:
-     ```bash
-     #!/bin/sh
-
-     errorExit() {
-        echo "*** $*" 1>&2
-        exit 1
-     }
+At the end of this step, try `ssh`ing between nodes to ensure success.
 
-     curl --silent --max-time 2 --insecure https://localhost:6443/ -o /dev/null || errorExit "Error GET https://localhost:6443/"
-     if ip addr | grep -q <VIRTUAL-IP>; then
-        curl --silent --max-time 2 --insecure https://<VIRTUAL-IP>:6443/ -o /dev/null || errorExit "Error GET https://<VIRTUAL-IP>:6443/"
-     fi
-     ```
-
-    Replace the `<VIRTUAL-IP>` by your chosen virtual IP.
+{{% capture steps %}}
 
-4. Restart keepalived. While no Kubernetes services are up yet it will log health check fails on all master nodes. This will stop as soon as the first master node has been bootstrapped.
-{{% /tab %}}
-{{< /tabs >}}
+# External Etcd -- The instructions
 
+This guide assumes calico will be used as the pod network and will therefore use
+some default values, but they will be called out explicitly so replacing calico
+with another pod network is possible.
```

Comment:
What about calling this out in a note in the prerequisites? Then we could get rid of the comment about Calico in the Pod CIDR step, too. The note would look something like:

"The following examples run Calico as the Pod networking provider. If you run another networking provider, make sure to replace any default values as needed."